src/py/flwr/server/superlink/fleet/grpc_bidi/grpc_server.py

```diff
@@ -218,6 +220,9 @@ def generic_create_grpc_server(  # pylint: disable=too-many-arguments
     server : grpc.Server
         A non-running instance of a gRPC server.
     """
+    # Check if port is in use
+    _is_port_in_use(server_address)
+
```

Comment:
This should be used in other places as well:
- SuperExec API
- SuperLink, Driver API
- SuperLink, Fleet API gRPC rere
- SuperLink, Fleet API REST
- SuperNode, ClientAppIo API

Reply:
It's the generic_grpc_server so it's applied to all places as well.